src/ast/ast.cpp

```diff
@@ -633,7 +618,7 @@ Map::Map(const Map &other) : Expression(other)
 }
 
 FieldAccess::FieldAccess(const FieldAccess &other)
-    : Expression(other), expr(nullptr)
+    : Expression(other) // TODO why not copy expr??
```

Comment:
@fbs Do you remember if `expr` wasn't copied for a reason?

Reply:
nope :(